consumer-test/runTest.sh

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env bash
 
-EXAMPLE_PROJECT_BRANCH=$1
+TEST_CASE=$1
+TEST_CASE_ROOT=$2
+TEST_CASE_WORKSPACE="${TEST_CASE_ROOT}/workspace"
 
 LIBRARY_VERSION_UNDER_TEST=$(git log --format="%H" -n 1)
 REPOSITORY_UNDER_TEST=${TRAVIS_REPO_SLUG:-SAP/jenkins-library}
 
-rm -rf workspace
-git clone -b "${EXAMPLE_PROJECT_BRANCH}" https://github.com/sap/cloud-s4-sdk-book workspace
-cp -f ../jenkins.yml workspace
-cd workspace || exit 1
+git clone -b "${TEST_CASE}" https://github.com/sap/cloud-s4-sdk-book "${TEST_CASE_WORKSPACE}"
```

Comment:
It think not all examples will go into this repository. Thus we could already put the branch name and the repository ulr into the yml file. But also fine in a follow-up when we need it.

Reply:
I know, I know ...
Basically this was what I had in mind when I wrote the comments

>\# Empty for the moment.
>\# Might contain test configuration in the future.

in the files located under `consumer-test/testCases/s4sdk/`.

I would like to propose to change that in a follow-up.